src/components/partnership/dto/update-partnership.dto.ts

```diff
@@ -25,10 +25,10 @@ export abstract class UpdatePartnership {
   readonly mouStatus?: PartnershipAgreementStatus;
 
   @DateField({ nullable: true })
-  readonly mouStart?: CalendarDate;
+  readonly mouStartOverride?: CalendarDate;
 
   @DateField({ nullable: true })
-  readonly mouEnd?: CalendarDate;
+  readonly mouEndOverride?: CalendarDate;
```

Comment:
These same changes should be made in `CreatePartnership` as well.

Reply:
Updated